<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneForm</class>
 <widget class="QWidget" name="SceneForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="videoTable">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addVideoButton">
         <property name="text">
          <string>Add Video</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteVideoButton">
         <property name="text">
          <string>Delete Video</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="doneButton">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
